understanding/20/images-of-text.html

```diff
@@ -44,6 +44,10 @@ <h2>Intent of Images of Text</h2>
       <p>Images of text can also be used where it is possible for users to customize the image
          of text to match their requirements.
       </p>
+
+      <p>If it is not possible to replace an image of text with actual text or to allow users to customize the image, the author must provide an [accessible alternative](https://www.w3.org/TR/WCAG22/#cc2) by duplicating the meaning or content of the image in text form. 
+         The alternative could replicate the text verbatim. If converting the visual image to text results in a loss of meaning or readability, the author could add or alter some words to convey the style, emphasis, or any other meaning that was lost.
+      </p>
```

Comment:
Based on yesterday's discussion, I have drafted a new proposal with a different approach to avoid any ambiguity or conflict with the normative portion of the SC:

Proposal:
```suggestion
      <p>An image of text accompanied by visible text that replicates its content verbatim is exempt from this success criterion, as the image is not used to convey information but serves a decorative purpose, with the text providing the actual information.
Moreover, the text does not always need to replicate the image of text content exactly. Verbatim replication might result in a loss of meaning or readability. Authors can modify or add words to convey style, emphasis, or other meanings that could be lost in a direct replication.
      </p>
```

Reply:
I would actually suggest dropping this, and the previous change (lines 47-50), and maybe instead just adding something like the following (which I think is getting to the same point as we discussed):

```
<div class="note"><p>The Success Criterion is scoped to situations where <a>images of text</a> are used <em>rather than</em> text. If a page has an image of text, but also contains visible text that duplicates the textual content of the image and conveys the same meaning, then this Success Criterion does not apply.</p></div>
```

Reply:
I'm not sure that it doesn't apply, so much as that it meets the requirement.
If text is used, it's met. I'm wondering if we're able to align on some concept and wording like this:

> <p>Note: The Success Criterion is intended to address situations where <a>images of text</a> are used <em>rather than</em> text.  Where images of text are used <em>in addition to text</em> to convey the same information -- both are presented to the user --  this success criterion is met. This allows author to convey any stylistic information they desire, while also providing the information in text, whose presentation can then be manipulated by users to make it more distinguishable.

Reply:
I'm inclined not to include new text that would open the Pandora's box regarding _how much_ the text may deviate from the same information provided in the image of text before this SC is no longer met via this rationale.